src/style.css

```diff
@@ -0,0 +1,15 @@
+.inputvalue:focus {
+  outline: none;
+}
+
+.inputvalue.active {
+  outline: 0.5px black solid;
+}
+
+.todo-description {
+  border: none;
+}
+
+.todo-description:focus {
+  outline: none;
+}
```

Comment:
- [ ] consider grouping classes with the same styling( .inputvalue, .todo-description ) ` outline: none;` to reduce code repeat and make your code more presentable